### PR TITLE
Sticky share icons

### PR DESCRIPTION
**Pull request by stephanfowler**
Animate in the lower icons, fixed to the bottom of the page, once the top ones have scrolled off the top of the page. Like Buzzfeed does :-)

Initially a zero audience AB, for easier device testing.

sticky-shares

### CODE REVIEW

**Comment by johnduffell** [comment]
looks very pretty! =) :+1:

File: static/src/javascripts/projects/common/modules/ui/sticky-social.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                }
+            });
+
+        } else {
+            setStickiness();
+        }
+    }
+
+    function makeRevealable() {
+        bottomEl.addClass(stickyRevealableClassName);
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/ui/sticky-social.js]: fastdom?

**Comment by sndrs** [comment]
thank goodness for https://gist.github.com/jashkenas/2e00d2208a270600a28a :)

:+1:

File: static/src/javascripts/projects/common/modules/ui/sticky-social.js (hunk 1 above)
Comment (stephanfowler) [inline static/src/javascripts/projects/common/modules/ui/sticky-social.js]: Good catch

**Pull request merged**